ffw/ButtonsRPC.js

```diff
@@ -179,44 +179,72 @@ FFW.Buttons = FFW.RPCObserver.create(
           break;
         }
         case 'Buttons.SubscribeButton': {
-          try {
-              const params = request.params;
-              const appID = params.appID
-              const resultCode = this.subscribeButton(appID, params.buttonName);
-              console.log("Button " + params.buttonName + " " + resultCode + " resultCode");
+          const buttonName = request.params.buttonName;
+          const appID = request.params.appID;
```

Comment:
Maybe it's better to destructure `buttonName` & `appId`?

Reply:
@YuriiYefimenko Agree. Fixed in aba78752fc7a0d8058e1de1c10052b5fc2687c17